docs/application/native/guides/ui/dali/videoview.md

```diff
@@ -108,6 +110,19 @@ newMap.Insert( "volumeRight", 0.5f );
 mView.SetProperty( VideoView::Property::VOLUME, newMap );
 ```
 
+To change the display mode, use the `PLAYER_DISPLAY_MODE` enums (in [mobile](https://developer.tizen.org/dev-guide/3.0.0/org.tizen.native.mobile.apireference/group__CAPI__MEDIA__PLAYER__DISPLAY__MODULE.html) and [wearable](https://developer.tizen.org/dev-guide/3.0.0/org.tizen.native.wearable.apireference/group__CAPI__MEDIA__PLAYER__DISPLAY__MODULE.html) applications) .
```

Comment:
We don't provide the links to https://developer.tizen.org anymore.

Please see API Reference links in this page and change this sentence like other links.
For instance, we are using like below:
```
See the Player API Reference (in [mobile](../../../api/mobile/latest/group__CAPI__MEDIA__PLAYER__MODULE.html) and [wearable](../../../api/wearable/latest/group__CAPI__MEDIA__PLAYER__MODULE.html) applications).    
```